redirects.yaml

```
@@ -56,6 +56,12 @@ aboutus/privacypolicy/?: "/legal/data-privacy"
 aboutus/privacypolicy/thirdparties/?: "/legal/data-privacy"
 aboutus/trademarkpolicy.*/?: "/legal/intellectual-property-policy"
 advantage/?: "/pricing"
+ai/?: "/kubeflow"
```

you could use a pattern match here, but please test
```suggestion
ai(?P<page>/.*)/?: "/{page}"
```